k8s/namespaces/divorce/div-dn/demo.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   releaseName: div-dn
   values:
     nodejs:
-      image: hmctspublic.azurecr.io/div/dn:prod-d226e16-20220801162731 #{"$imagepolicy": "flux-system:demo-div-dn"}
+      image: hmctspublic.azurecr.io/div/dn:pr-635-710f5cd-20220810092258 #{"$imagepolicy": "flux-system:demo-div-dn"}
```

Comment:
you need to update actual demo-div-dn imagePolicy to new PR reference
https://github.com/hmcts/cnp-flux-config/blob/master/apps/divorce/div-dn/demo-image-policy.yaml#L9 

In this case you have, is the imagAutomation not working?

Reply:
It doesn't seem to be, no.  We're not seeing our PR's in Demo.